### ISSUE TITLE
A Dashboard icon would be great

### ISSUE THREAD

**Issue by n00b12345** [mrmn2/PdfDing#39]
A dashboard icon for adding PDFDing to homepage dashboards would be great to have.

This is the repo - https://github.com/walkxcode/dashboard-icons/

**Comment by mrmn2** [comment]
Yes that would be great. However, I am not really happy with the current logo as it is only meant as a temporary solution. Problem is I cannot really design a logo. I guess I'll create an issue asking for help in doing so.

**Comment by mrmn2** [comment]
Hi @n00b12345, I have done a bit of work regarding the icon. It is good enough for me and I have thus closed #42. Feel free to have the new logo added to the dashboard icons repo. You can find the new logo [here](https://github.com/mrmn2/PdfDing/blob/master/pdfding/static/images/logo_with_circle.svg). I guess this issue can now be closed.
